integration-tests/src/fees.rs

```diff
@@ -3,35 +3,85 @@
 use crate::kusama_test_net::*;
 use basilisk_runtime::TransactionPayment;
 use frame_support::weights::GetDispatchInfo;
+use primitives::nft::ClassType;
 use sp_runtime::codec::Encode;
 use xcm_emulator::TestExt;
 
+macro_rules! assert_eq_approx {
+	( $x:expr, $y:expr, $z:expr, $r:expr) => {{
+		let diff = if $x >= $y { $x - $y } else { $y - $x };
+		if diff > $z {
+			panic!("\n{} not equal\nleft: {:?}\nright: {:?}\n", $r, $x, $y);
+		}
+	}};
+}
+
+//NOTE: rust encoded call size is differen from UI encoded call size that's why we have asserts for 2 fees.
+
 #[test]
-// #[ignore]
-//TODO: fix this, fees calculcated in this test are apporx 2bsx off.
 fn transfer_transaction_fees() {
```

Comment:
I would improve the test name according to our ShouldWhen test naming template. ;)

Also applies for the other tests in this file.